InnovativeCars/force-app/main/default/classes/SalonExternal.cls

```diff
@@ -0,0 +1,112 @@
+/**
+* @author Agnieszka Ząbkowicz
+* @date 11.06.2021
+* @description class for wrapping showrooms and unwrapping external showrooms
+*/
+global class SalonExternal {
+    public SalonExternal() {
+        
+    }
+    
+    public SalonExternal(Showroom__c inShowroom) {
+        salonId = inShowroom.Id;
+        name = inShowroom.Name;
+        adres = inShowroom.Address__c;
+        switch on inShowroom.Type__c {
+            when 'Personal Cars'{
+                typ = 'personal_cars';
+            }
+            when 'Specialist Cars'{
+                typ = 'specialist_cars';
+            }
+            when 'All'{
+                typ = 'all';
+            }
+        }
+        accountId = inShowroom.Account__c;
+        strona = inShowroom.Website__c;
+        switch on inShowroom.Status__c {
```

Comment:
could this be done with map?
Map <String,String> statusToExternalStatus .... with values 'Open', 'open' etc
then:
status = statusToExternalStatus.get(inShowroom.Status__c)